time timestamp
int8 nodeID
detected_ball_data[] balls